File: prime_numbers/primes.ts
// show a list of prime numbers in typescript

// tell if a number is prime
function is_prime(num: number){
    var i:number;
    for (i=2; i<num-1; i++) {
        if (num % i == 0){
            return false
        }
    }
    return true;
}

// list the prime numbers
var i:number;
for (i=1; i<99; i++){
    if (is_prime(i)){
        console.log(i);
    }
}

File: prime_numbers/primes.py
# show a list of prime numbers
def is_prime(number):
    for i in range(2, number):
        if (number % i) == 0:
            return False 
    return True

if __name__ == "__main__":
    for i in range(2, 99):
        if is_prime(i):
            print(i)
